From 20dc46b6bc6c049787412f077f2877d169d29b00 Mon Sep 17 00:00:00 2001
From: Lukasz Majewski <lukma@denx.de>
Date: Wed, 10 Mar 2021 11:58:53 +0100
Subject: [PATCH 07/15] dts: Change pinmux configuration for ENET2_RX_ER pin
 (to be GPIO)

In the new HW setup - the ENET2_RX_ER pin needs to be configured as
GPIO with internal PULL down resistor (to avoid dropping frames).

Linux pinmux driver will not change configuration already set in
U-Boot.

Signed-off-by: Lukasz Majewski <lukma@denx.de>
---
 arch/arm/boot/dts/imx6ull-kie-inverter.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ull-kie-inverter.dts b/arch/arm/boot/dts/imx6ull-kie-inverter.dts
index 72b114e76a3a..bc26fc2490d0 100644
--- a/arch/arm/boot/dts/imx6ull-kie-inverter.dts
+++ b/arch/arm/boot/dts/imx6ull-kie-inverter.dts
@@ -264,7 +264,7 @@ pinctrl_enet2: enet2grp {
 			MX6UL_PAD_GPIO1_IO06__ENET2_MDIO	0x10010
 			MX6UL_PAD_GPIO1_IO07__ENET2_MDC		0x10010
 			MX6UL_PAD_ENET2_RX_EN__ENET2_RX_EN	0x1b0b0
-			MX6UL_PAD_ENET2_RX_ER__ENET2_RX_ER	0x1b0b0
+			MX6UL_PAD_ENET2_RX_ER__GPIO2_IO15       0x80000000
 			MX6UL_PAD_ENET2_RX_DATA0__ENET2_RDATA00	0x1b0b0
 			MX6UL_PAD_ENET2_RX_DATA1__ENET2_RDATA01	0x1b0b0
 			MX6UL_PAD_ENET2_TX_EN__ENET2_TX_EN	0x1b010
-- 
2.37.2

